src/transformers/generation/utils.py

```diff
@@ -1168,6 +1179,8 @@ def generate(
         synced_gpus: Optional[bool] = None,
         assistant_model: Optional["PreTrainedModel"] = None,
         streamer: Optional["BaseStreamer"] = None,
+        negative_prompt: Optional[torch.Tensor] = None,
```

Comment:
```suggestion
        negative_prompt_ids: Optional[torch.Tensor] = None,
```
(to be more specific -- and, if possible, this change would be propagated to all uses)